mesonbuild/compilers/vala.py

```diff
@@ -49,6 +49,9 @@ def get_compile_only_args(self):
     def get_pic_args(self):
         return []
 
+    def get_pie_args(self):
```

Comment:
why not add this to the `Compiler` class instead?

Reply:
really this is just a hold over, there's another series to add proper pie support to compilers, and I plan to rebase on that.